edenia/eden-spend-explorer
# fix(webapp): download button and titles

### fix Webapp

### What does this PR do?

- Resolve #186 
- Resolve #187  

### Steps to test

1. Run Project
2. Must see this: 
![imagen](https://user-images.githubusercontent.com/49013556/197896806-f5809bb4-f90e-4027-a304-e237aa216422.png)

#### CheckList
